Tests/Controllers/PeopleController.cs

```diff
@@ -50,6 +51,13 @@ public Person PostPerson(string id, Person person)
         {
             person.Identifier = id;
             return person;
+        }
+
+        [HttpPost]
+        [Route("relationships/people")]
+        public JObject PostParams([FromBody] JObject bodyParams)
```

Comment:
I was actually using JObject, because we had to write a wrapper around EF, and our Create method takes either an anonymous object or a JObject to assign attributes. I could see if I could have it copy all the properties from the class. Right now my method signature is:

        public JToken Create([FromBody] TDbType bodyParams)